src/Microsoft.DotNet.Build.Tasks.Feed/FeedActionTask.cs

```diff
@@ -0,0 +1,136 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using MSBuild = Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System.Collections.Generic;
+using System.Diagnostics;
+
+namespace Microsoft.DotNet.Build.Tasks.Feed
+{
+
+    public class FeedActionTask : MSBuild.Task
+    {
+
+        [Required]
+        public string Task { get; set; }
```

Comment:
Task is overloaded term I think I would give it a better description. Is there any reason you don't just have two Tasks? One for Push and one for Configure?

Reply:
fixed.